pymc4/flow/executor.py

```diff
@@ -509,6 +537,26 @@ def proceed_distribution(
         state.distributions[scoped_name] = dist
         return return_value, state
 
+    def proceed_deterministic(
```

Comment:
Here
https://github.com/pymc-devs/pymc4/blob/4685b96604290b21a16275d90d3e33ed6bc740f1/pymc4/flow/executor.py#L588

We may have an output that requested to be recorded. I would treat it as deterministic too

Reply:
Output of the model

Reply:
Ok, so mean that I would replace this:
`state.untransformed_values[return_name] = return_value`
with this:
`state.deterministics[return_name] = return_value`, right?